etl/data_extraction/scraper/weltwaerts.py

```diff
@@ -0,0 +1,134 @@
+            'geo_location': {
+                'lat': lat,
+                'lon': lon,
+            } if lat and lon else None, # If longitude and latitude are None, geo_location is set to None
+            'languages': param_box.find('li').findNext('li').find('span', {'class': 'parameter__value'}).decode_contents().strip() or None,
+            'requirements': content.find('h2', text='Anforderungen an dich').findNext('div').p.decode_contents().strip() or None,
+        }
+
+        # Removing HTML-Tags
+        contact_raw = re.sub(r'</?p>', '', parsed_object['contact'])
+        contact_raw = re.sub(r'<br/?>', '\n', contact_raw)
+        contact_raw = re.sub(r'<a.*</a>', '', contact_raw)
+        contact_raw = contact_raw.replace('</br>', '')
+        # Removing newlines
+        contact_raw = contact_raw.replace('\n\n', '\n').strip()
+        contact_split = list(filter(None, contact_raw.split('\n')))
+
+        # If the contact data contains additional information, it is combined into a string
+        if len(contact_split) > 3:
+            names_split = contact_split[:(len(contact_split)-2)]
+            names_raw = ', '.join(names_split)
+            contact_split = [names_raw, contact_split[-2], contact_split[-1]]
+
+        parsed_object['post_struct'] = {
+            'title': parsed_object['title'],
+            'categories': parsed_object['categories'],
+            'location': {
+                'country': parsed_object['location'].split(', ')[-1] or None,
+            },
+            'task': re.sub(r'</?p>', '', parsed_object['task']).strip(),
+            'timing': parsed_object['timing'],
+            'organization': {
+                'name': parsed_object['organization'],
+            },
+            'contact': {
+                'name':  contact_split[0].strip() or None,
+                'zipcode': contact_split[2][:5].strip() or None,
+                'city': contact_split[2][5:].strip() or None,
+                'street':  contact_split[1].strip() or None,
+                # Extract the e-mail address, if available
+                'email': contact.find('a', href=re.compile("mailto:.*"))['href'].replace('mailto:', '').strip() or None,
+            },
+            'link': parsed_object['link'],
+            'source': parsed_object['source'],
+            'image': parsed_object['image'],
+            'geo_location': parsed_object['geo_location'],
+            'languages': parsed_object['languages'],
+            'requirements': parsed_object['requirements'],
+        }
+
+        return parsed_object
+
+    #   Adds all URLs of detail pages, found on the search pages, for the crawl function to scrape
+    def add_urls(self):
+        import time
+
+        search_page_url = f'{self.base_url}/de/ep-ergebnis.html'
+        next_page_url = search_page_url
+
+        index = 1
+        while next_page_url:
+
+            response = self.soupify(next_page_url)
```

Comment:
Currently you have only 10 results per page. The page gives you an option to scrape up to 50 results per page. This is possible by sending this arument with the request. Is there a possible implementation? (Maybe by overloading the soupify method which takes and request argument)
![grafik](https://user-images.githubusercontent.com/64203299/100658645-b4948f80-334f-11eb-90d4-5d74551d0738.png)